src/SignalR/server/Core/src/Internal/HubClients`T.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.AspNetCore.SignalR.Internal
 {
-    internal class HubClients<THub, T> : IHubClients<T> where THub : Hub
+    internal class HubClients<THub, T> : HubClientsBase<T> where THub : Hub
```

Comment:
This class the reason you didn't want to name HubClientsBase HubClients? Just rename this one to DefaultHubClients. It's internal and in another namespace so the HubClients name isn't burned. We should use it for the public type.